modules/rostests/kmtests/kmtest/service.c

```diff
@@ -101,6 +102,102 @@ KmtCreateService(
                              ServiceHandle);
 }
 
+/**
+ * @name KmtGetServiceStateAsString
+ *
+ * @param ServiceState
+ *        Service state as a number
+ *
+ * @return Service state as a string
+ */
+static
+PCSTR
+KmtGetServiceStateAsString(
```

Comment:
This seems overkill for a trace message in an error case. I'd say people can look up the numbers if they come by that unexpected case.